axisregistry/wonkiness.textproto

```diff
@@ -1,12 +1,12 @@
 # WONK based on https://github.com/undercasetype/Fraunces#variable-axes
 tag: "WONK"
-display_name: "Wonkiness"
+display_name: "Wonky"
 min_value: 0
 max_value: 1
 default_value: 0
 precision: 0
 fallback {
-  name: "Normal"
+  name: "NonWonky"
```

Comment:
Spaces to match others, e.g. "Extra Light" not "ExtraLight."

Reply:
Might want "Not Wonky" over "Non Wonky"?

Reply:
@thundernixon, does this change (noT instead of Non) would require to regenerate the fonts with a new stat table ie. go through the all process again ?

Reply:
The space is mandatory so if that's true you can't get out of going through the process again.

Reply:
In the fonts we send, we don't have space for any fallback names. So the space would be added only in the axis registry. Although a difference of letter may be a bigger inconsistency.

Reply:
I’m okay to rebuild the fonts to change this if needed, once we land on a solution. 🙂

Hmm, as for "Not Wonky" vs "Non Wonky," I think it could probably go either way. I gravitate towards "Non Wonky" because non- is a prefix, and in my head this binds it better as a single style descriptor, whereas "Not Wonky" sounds more like a statement or near-sentence. I can’t really decide whether there is a meaningful difference between "The font style is nonwonky" and "The font style is not wonky." To me, using specific style terms as single words does help suggest that they are specific values, rather than generalized statements. E.g. "The font weight is SemiBold" indicates a specific wght of 600, whereas "The font weight is Semi Bold" suggests that the weight is somewhere less than 700. 

Perhaps, though, an important thing I got wrong in "NonWonky" is that it mistreats the prefix. It should probably be a choice between "Not Wonky" and "Nonwonky." I somewhat lean to the latter, but I don’t have a strong feeling one way or the other. I just want to get this font out to The People, and will accept any decision. :)